### DISCUSSION TITLE
image creation is verry slow

### DISCUSSION THREAD

**Discussion by steto123** [seruva19/kubin#126] [Q&A]
I have installed this nice webui to use kandinsky 2.2 and after some times it works. but really slow. I have an 3080TI on my PC and it runs normally a few seconds for generating an image (for example using sd.next and SDXL).  For kandinsky it runs some minutes for a image and i  think this is a configuration problem. What should i enable in diffusers settings? Only half precision weights is enabled at this time.

Many Thanks

**Comment by seruva19** [comment]
Hi.
You can try activating 'Enable prior generation on CPU'. 
Another option is enabling the following flags 'Enable sliced attention' and 'Enable sequential CPU offload' (but do not combine them with 'Enable prior generation on CPU', it most likely won't work for now).

I don't use SD.Next, does Kandinsky there works fast out of the box for you?

**Comment by steto123** [comment]
Many thanks. I will check this. But since a hour kandinsky 2.1 is running via script in my sd.next installation.  But it is also really slow compared to SDXL 1.0.
